sql/2021/pwa/manifests_not_json_parsable_sw.sql

```diff
@@ -0,0 +1,40 @@
+#standardSQL
+# Manifests that are not JSON parsable for service worker pages
```

Comment:
Same observations as the previous one. Here we could also consider adding the `$._pwa.serviceWorkerHeuristic`.